docs/articles/documentation/test-api/assertions/index.md

```diff
@@ -20,9 +20,14 @@ This topic contains the following sections.
 
 ## Assertion Structure
 
-TestCafe assertions start with the `expect` method exposed by [test controller](../test-code-structure.md#test-controller),
-which accepts the `actual` value argument. Then, the assertion method follows. An assertion method accepts an expected value
-and optionally other arguments. For example, this is how the deep equality assertion looks like.
+TestCafe assertions start with the `expect` method exposed by [test controller](../test-code-structure.html#test-controller).
+This method accepts the actual value. You can pass an object, a [Selector's DOM node state property](../selecting-page-elements/selectors.md#define-assertion-actual-value)
+or a [client function](../obtaining-data-from-the-client.md).
```

Comment:
Maybe `Client Function` instead of `client function` since it's our term, but not a regular client function?